pkg/subctl/cmd/join.go

```diff
@@ -85,6 +86,7 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
 	cmd.Flags().StringVar(&brokerClusterContext, "broker-cluster-context", "", "Broker cluster context")
+	cmd.Flags().StringVar(&cableDriver, "cable-driver", "", "Cable driver implementation")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&cableDriver, "cable-driver", "strongswan", "Cable driver implementation")
```

Reply:
We need a default

Reply:
The default is decided in the backend when empty string is passed. As per my previous comment:

> I think we should specify "" default value and let the engine backend decide what default driver to use. That way if we change the default driver in the backend in a future release, we don't have to change it here and upgrades will automatically pick up the new default.

Reply:
Ok, makes sense. Migration from one driver to another will need to be checked. We will also need to warn the customer of the change of driver when that's about to happen.